lib/api/bucketPutCors.js

```diff
@@ -33,14 +33,10 @@ function bucketPutCors(authInfo, request, log, callback) {
         return callback(errors.MissingRequestBodyError);
     }
 
-    if (request.headers['content-md5']) {
-        const md5 = crypto.createHash('md5')
-            .update(request.post, 'utf8').digest('base64');
-        if (md5 !== request.headers['content-md5']) {
-            log.debug('bad md5 digest', { error: errors.BadDigest });
-            monitoring.promMetrics('PUT', bucketName, 400, 'putBucketCors');
-            return callback(errors.BadDigest);
-        }
+    const err = validateChecksumsNoChunking(request.headers, request.post);
+    if (err && err.error !== ChecksumError.MissingChecksum) {
```

Comment:
If I understand correctly, if the error is `MissingChecksum`, ie the client sent a body but not the content-md5 header, we just ignore any validation?